app/api/helpers/scheduled_jobs.py

```diff
@@ -155,6 +156,16 @@ def expire_pending_tickets():
         db.session.commit()
 
 
+def delete_ticket_holders_no_order_id():
+    from app import current_app as app
+    with app.app_context():
+        order_expiry_time = get_settings()['order_expiry_time']
+        db.session.query(TicketHolder).filter(TicketHolder.order_id == None,
```

Comment:
Use `TicketHolder.query.filter_by`

Reply:
any specific reason for it?

Reply:
Consistent style, shorter

Reply:
@iamareebjamal For comparison I of created_at+current_time, I still have to use filter as it wont work in filter_by

Reply:
OK

Reply:
@iamareebjamal then shall I change it to combination of filter and filter_by or is it good to go?

Reply:
Use `TicketHolder.query.filter`